Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -12,6 +12,9 @@ import Foundation
 @Observable
 final class SettingsViewModel {
     let authenticationViewModel: AuthenticationViewModel
+    let privacyText = "Privacy Policy"
```

Comment:
I don't think we need to make this a variable, it's fine to just have `Privacy Policy` directly in the view
This should also change the `Localizable.xcstrings` file

Reply:
Corrected

Reply:
@Oksenoyt Next time, just go ahead and click "resolve" because that shows me you did address the comment, as opposed to me having to go through and click resolve on each